src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,60 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: Callback;
+};
+
+type Callback = (tab: Tab) => void;
+
+export const Tabs: React.FC<Props> = (props: Props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const tabContent = tabs.find(tab => tab.id === selectedTabId);
+
+  return (
+    <>
+      <div
+        className="tabs is-boxed"
+      >
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              className={classNames(
+                {
+                  'is-active': tab.id === selectedTabId
+              || (!tabContent && tab === tabs[0]),
+                },
+              )}
+            >
+              <a
+                href={`#${tab.id}`}
+                onClick={() => {
+                  if (selectedTabId !== tab.id) {
```

Comment:
Let's move this statement to the `onTabSelected` function

Reply:
with replacing this statement inside `onTabSelected` test #7 failed:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/106744410/185398167-170a71ff-765f-4eff-b1c5-975b29507a1d.png">